app/views/email/default_template.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="%{html_lang}" xml:lang="%{html_lang}">
+
+<head>
+  <meta http-equiv="Content-type" name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=no, width=device-width">
+  <title></title>
+</head>
+
+<body>
+
+<!--[if mso]>
+<style type="text/css">
+body, table, td, th, h1, h2, h3 {font-family: Helvetica, Arial, sans-serif !important;}
+</style>
+<![endif]-->
+
+%{email_content}
+
+<!-- prevent Gmail on iOS font size manipulation -->
```

Comment:
I know you didn't add this but :scream:

Reply:
Yeah there were a lot of discoveries about html rendering in email clients while working on the summary email...